src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,64 @@
+.calendar {
+  width: 7.1 * $box-width;
+  height: 5.1 * $box-height;
```

Comment:
why are you using float values like `7.1`? maybe it would be better to multiply on 7 and add exact `padding`/`margin`/`gap`?